stdlib/asm/crypto/dsa/rpo_falcon512.masm

```diff
@@ -7,6 +7,8 @@ const.J=77321994752
 const.M=12289
 const.SQUARE_NORM_BOUND=34034726
 
+const.EVENT_FALCON_SIG_TO_STACK=3419226139
```

Comment:
How did we choose this value?

Reply:
It was chosen randomly, like the other [system events](https://github.com/0xPolygonMiden/miden-vm/blob/cdbfaed94de2d06976e7eac10f956186f2f6c8a5/core/src/sys_events.rs#L13-L30). This basically makes collisions with user-defined events improbable.